README.md

```diff
@@ -273,22 +275,27 @@ It is also recommended but not required that each bot have a unique command pref
 
 ## Appendix
 
-### config.yaml parameter descriptions
+### Parameter Descriptions for config.yaml
 
-#### Core bot parameters
+#### Core Bot Parameters
 
--   **name** - A name for the bot. It can be unique if you want, but is not required. However it will be more difficult to distingush multiple bots if you are running multiple.
--   **token** - A discord developer API token. You can get one [here](https://discord.com/developers/applications)
--   **command_prefix** - A single character which you will use to preface all commands for this bot e.g. !music
--   **logging** - Enable or disable logging for this bot by setting the enabled value to True or False.
--   -   **logging_level *(Default=NOTSET)*** - The level of logging. Possible options: NOTSET, INFO, WARNING, ERROR, DEBUG
+|Parameter|Default Value(s)|Description|
+|---|:---:|---|
+|name|`null`|A name for the bot. It can be unique if you want, but is not required. However it will be more difficult to distingush multiple bots if you are running multiple.|
+|token|`null`|A discord developer API token. You can get one [here](https://discord.com/developers/applications)
+|command_prefix|`"!"`|A single character which you will use to preface all commands for this bot e.g. `!music`|
```

Comment:
Can you add a note that some characters such as the ! required double quotes, so you should always surround your command prefix with double quotes to be safe.

Reply:
Note added. DONE!